.github/workflows/builder_go_slsa3.yml

```diff
@@ -372,6 +372,7 @@ jobs:
         if: inputs.upload-tag-name != ''
         with:
           tag_name: "${{ inputs.upload-tag-name }}"
+          prerelease: ${{ inputs.prerelease }}
```

Comment:
NOTE: updated because the Action code updates the prerelease tag in all cases https://github.com/softprops/action-gh-release/blob/master/src/github.ts#L259

Reply:
Thanks! I had assumed in doing the review that it would preserve this. Great check.

Reply:
just in case...